lib/auth/new_web_user.go

```diff
@@ -43,6 +43,14 @@ import (
 // CreateSignupToken creates one time token for creating account for the user
 // For each token it creates username and otp generator
 func (s *AuthServer) CreateSignupToken(userv1 services.UserV1, ttl time.Duration) (string, error) {
+	clusterConfig, err := s.GetClusterConfig()
+	if err != nil {
+		return "", trace.Wrap(err)
+	}
+	if clusterConfig.GetLocalAuth() == false {
```

Comment:
I think this condition can be omitted here, the logic of creating a signup token does not need to be aware of this detail.